docs/source/eks.md

```diff
@@ -82,28 +82,30 @@ For this guide, we'll create an EKS cluster with the following:
       allow: true
 ```
 
-### Installing Required Tools
+### Prerequisites
 
 #### Installing script third party dependencies
 
 Script requires several dependencies:
-- Helm - See: https://docs.helm.sh/using_helm/#installing-helm
 - eksctl - See: https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html
 - kubectl - See: https://kubernetes.io/docs/tasks/tools/install-kubectl/
 
+#### Setting up local nodes with NodeConfig
 
-#### Install the local provisioner
+To run Scylla, it is most important, and necessary in production, that you convert the ephemeral storage's filesystem to `xfs`.
```

Comment:
```suggestion
To run ScyllaDB, it is necessary in production, that you format your storage used by ScyllaDB to XFS filesystem.
```
I wouldn't emphasise that much

Reply:
The most important part actually comes from Scylla docs: https://opensource.docs.scylladb.com/stable/getting-started/system-configuration.html#set-up-raid-and-filesystem. What's wrong with the other sentence?

Reply:
We could phrase it like 
```suggestion
 ScyllaDB, except for developer mode, requires storage with XFS filesystem. The local NVMes from the cloud provider usually come as individual devices. To use their full capacity together, you'll first need to form a RAID array from those disks.
```

Reply:
done